WalletWasabi.Backend/InitConfigStartupTask.cs

```diff
@@ -19,10 +18,10 @@ public class InitConfigStartupTask : IStartupTask
 		public WebsiteTorifier WebsiteTorifier { get; }
 		public Global Global { get; }
 
-		public InitConfigStartupTask(Global global, IHostingEnvironment hostingEnvironment)
+		public InitConfigStartupTask(Global global, IWebHostEnvironment hostingEnvironment)
 		{
 			Global = global;
-			WebsiteTorifier = new WebsiteTorifier(((HostingEnvironment)hostingEnvironment).WebRootPath);
```

Comment:
`HostingEnvironment` came from `Microsoft.Extensions.Hosting.Internal`, which doesn't exist anymore. `IWebHostEnvironment` with `WebRootPath` seems to compile fine. @NicolasDorier just to be sure, this change is correct, right?

Reply:
So in 3.0 I use `IWebHostEnvironment` instead when I migrated NBXplorer, worked without problem.

Reply:
You want to read https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30

Reply:
Thanks!